backend/app/assets/javascripts/spree/backend/cms/links.es6

```diff
@@ -0,0 +1,24 @@
+document.addEventListener('DOMContentLoaded', function() {
+  const linkSwitcher = $('.link_switcher').select2()
```

Comment:
AFAIK we shouldn't be using jQuery anymore as we want to stop using it in spree 5 entirely.
@damianlegawiec am I correct?

Reply:
@mad-eel we should avoid, however unfortunately `select2` requires jquery - we would need to think how to resolve in the future (change the library, check select2 maintainers plans for future development and dropping jquery dependency and so on)

Reply:
Hi @mad-eel 

Select2 is a jQuery based lib, if we are using select2 when it move to V5 then I can re-write all this in plane JS